app/javascript/forms/nuage-cloud-subnet-form.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { Form, Field, FormSpy } from 'react-final-form';
+import { Form as PfForm, Grid, Button, Col, Row, Spinner } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { required } from 'redux-form-validators';
+
+import { FinalFormField, FinalFormTextArea, FinalFormSelect } from '@manageiq/react-ui-components/dist/forms';
+import { ip4Validator, netmaskValidator } from '../utils/validators'
+
+class NuageCloudSubnetForm extends Component {
+  render() {
+    if(this.props.loading){
+      return (
+        <Spinner loading size="lg" />
+      );
+    }
+
+    return (
+      <Form
+        onSubmit={() => {}} // handled by modal
+        render={({ handleSubmit }) => (
+          <PfForm horizontal>
+            <FormSpy onChange={state => this.props.updateFormState({ ...state, values: state.values })} />
+            <Grid fluid>
+              <Row>
+                <Col xs={12}>
+                  <Field
+                    name="name"
+                    component={FinalFormField}
+                    label="Name"
```

Comment:
You should use i18n for all strings. `name={__('Name')}` should work. `__` (two underscores) should be available on window.